### PR TITLE
[IMP] Note module demo data adjustment

### PR DESCRIPTION
**Pull request by tsabi**
There is a check/todo list functionality in note module. Lets provide some of the demo notes to be a check list.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tsabi** [comment]
@pedrobaeza Hi, you old me not to create ticket for PRs. You see, this is more than two month old, no one take care.

**Comment by pedrobaeza** [comment]
Yes, but the same may happen with PRs: there's no guarantee that someone sees and merges this (or fixes in the case of an issue), as it's based in a best effort basis. Putting a PR makes more chances and initially the work is done, but not being critical issues, the importance/priority of them is perceived as low. The only thing I can do is to ping someone to see if they can check.

@Gorash you appear as suggested reviewer. Maybe you take a look.

**Comment by Gorash** [comment]
Hello, for my part I have been on a completely different project for a long time. I don't know the guidelines for these data demos. So I don't know how to give a favorable opinion for that.

**Comment by pedrobaeza** [comment]
Thanks for the followup, @Gorash, then we let this until someone check it. Maybe @mart-e can redirect this to the proper person.

File: addons/note/data/note_demo.xml [hunk 1]
```diff
@@ -49,21 +52,16 @@
...
-      </p><p>* Tower
-      silent, better when watching films
-      </p><p>* Blueray drive ?
-      is it interesting yet ?
-      </p><p>* Screen
-      a big one, full of pixels, of course !</p>]]>
+     <p><br/></p>
+     <ul class="o_checklist">
+     <li id="checklist-id-1"><p>Motherboard according to processor </p></li>
+     <li id="checklist-id-2" class=""><p>Processor need to decide </p></li>
```
Comment (mart-e) [inline addons/note/data/note_demo.xml:58]: why is there an empty class here

**Comment by tsabi** [comment]
@mart-e Hello! Merged your comments, and squashed my commits to 1.

**Comment by mart-e** [comment]
Thanks for the update

@robodoo r+

**Pull request closed without merging**